### PR TITLE
Got a pretty good looking grid going

### PR DESCRIPTION
**Pull request by vacas5**

### CODE REVIEW

File: src/CharacterList.js [hunk 1]
```diff
@@ -19,17 +19,45 @@ class CharacterList extends Component {
...
-        {this.state.characters.map( (character, index) => {
-          return <li key={index}>{character.name}, {character.description}</li>;
-        })}
-      </ul>
+      <div>
+        <h2 className="main_title">Characters</h2>
+        <div className="cards">
+          {this.state.characters.map( (character, index) => {
+            return (
+              <div className="card" key={character.id}>
```
Comment (nicrocs) [inline src/CharacterList.js]: If we are going to reuse this card div for the other components (like series), we should probably make it it's own component. <Card data={character} type="character"> or something to that effect

File: src/CharacterList.js [hunk 2]
```diff
@@ -19,17 +19,45 @@ class CharacterList extends Component {
...
     return (
-      <ul className="character_list">
-        {this.state.characters.map( (character, index) => {
-          return <li key={index}>{character.name}, {character.description}</li>;
-        })}
-      </ul>
+      <div>
+        <h2 className="main_title">Characters</h2>
+        <div className="cards">
+          {this.state.characters.map( (character, index) => {
```
Comment (nicrocs) [inline src/CharacterList.js]: if your key is the character.id, you probably don't need to pass along the index anymore

**Comment by vacas5** [comment]
I think that makes sense. Can we merge and then I'll refactor in a separate branch?

**Comment by nicrocs** [comment]
works for me

**Pull request merged**